### ISSUE TITLE
bug: navbar missing properties on mobile and tablet

### ISSUE THREAD

**Issue by RinorG** [ory/web#650]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/web/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/web/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

The navigation menu in mobile and tablet doesn't show the same options as desktop does.

### Reproducing the bug

Open in desktop.
Open in mobile.
Compare.

### Version

prod

**Comment by BlueAcidFrog** [comment]
It all speaks for a redesign of web :)

**Comment by aeneasr** [comment]
Yes, they use separate navigation sources. We decided to do it that way initially to make the mobile navigation easier because there is too many items.

Desktop navigation:

https://github.com/ory/web/blob/56dd7fddfd1dce804cde5419577a239caae31e2b/src/page-content/navigation/header.json#L2

Mobile navigation:

https://github.com/ory/web/blob/56dd7fddfd1dce804cde5419577a239caae31e2b/src/page-content/navigation/header.json#L121

You can edit those files and bring the two in line. But please keep in mind that you have much less space on mobile and we need to focus on the important items in it!
